client/homepage/stats-overview/style.scss

```diff
@@ -0,0 +1,46 @@
+.woocommerce-stats-overview {
+	.woocommerce-card__body {
+		padding: 0;
+	}
+	.woocommerce-summary {
+		margin: 0;
+	}
+}
+
+.woocommerce-stats-overview__more-btn {
```

Comment:
This may be something you were planning on addressing in the follow-up, but one very minor styling issue is this focus on the more button.

<img width="565" alt="Screen Shot 2020-05-07 at 8 42 19 PM" src="https://user-images.githubusercontent.com/10561050/81327186-8bef3480-90a3-11ea-9061-8b839dc8e8ec.png">

Reply:
Nice, I made the button `inline-block` so doesn't also take up the entire space in https://github.com/woocommerce/woocommerce-admin/pull/4288/commits/a5c007efd1c40f33e46d9dd15893447ada763b49 and this looks better when it has focus.

<img width="842" alt="Screen Shot 2020-05-08 at 10 42 07 AM" src="https://user-images.githubusercontent.com/1922453/81351939-7157a280-9119-11ea-8af5-d2e9a5c5a072.png">